File: AlPHA/Practice.java
import java.util.Scanner;

class Practice
{
  public static void main(String[] args) {
   // accept the number from user and check the number is divisible by 5

   Scanner sobj = new Scanner(System.in);

   System.out.println("Enter the number");
   int ino = sobj.nextInt();

   if(ino < 0)
   {
    System.out.println("Hello");
   }else{
    System.out.println("Demo");
   }
  }
}

File: Practice.c
#include<Stdio.h>

int main()
{ 
   int iNo = 0;
   printf("Enter the number  ");
   scanf("%d",&iNo);

   if(iNo % 2== 0)
   {
      printf("Number is Even");
   }
   else
   {
      printf("Number is Odd");
   }
}